<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiTransform</class>
 <widget class="QWidget" name="UiTransform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vLayout_main">
    <item>
     <widget class="QPushButton" name="btn_translate">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>&gt;平移</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_translate" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>85</height>
       </size>
      </property>
      <widget class="QLabel" name="label_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>平移x:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>       z:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>       y:</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>0</y>
         <width>116</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vLayoutTranslate">
        <item>
         <widget class="QLineEdit" name="lineEdit_translationX">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_translationY">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_translationZ">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_rotate">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>&gt;旋转</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_rotate" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>85</height>
       </size>
      </property>
      <widget class="QLabel" name="label_33">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>旋转x:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_30">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>       z:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>       y:</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>0</y>
         <width>116</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vLayoutRotate">
        <item>
         <widget class="QLineEdit" name="lineEdit_rotationX">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_rotationY">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_rotationZ">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_scale">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>&gt;缩放</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_scale" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>85</height>
       </size>
      </property>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>       y:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_34">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>缩放x:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_35">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>       z:</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>0</y>
         <width>116</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vLayoutScale">
        <item>
         <widget class="QLineEdit" name="lineEdit_scaleX">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_scaleY">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_scaleZ">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
